### PR TITLE
Bugfix - Uris are being deserialized like strings, and that's bad

### PR DESCRIPTION
**Pull request by shiftkey**
Let me know if there's anything missing from the PR

### CODE REVIEW

File: src/SimpleJson.Tests/DeserializeObjectTests.cs [hunk 1]
```diff
@@ -496,6 +496,26 @@ public void DeserializeDoubleQuotesCorrectly()
...
+            Assert.AreEqual(new Uri("https://github.com/shiftkey/simple-json/issues/1"), result.url);
+        }
+
+        [TestMethod]
+        public void DeserializeInvalidUriCorrectly()
+        {
+            var json = "{\"url\":\"this is a broken uri\"}";
+            var result = SimpleJson.DeserializeObject<ClassWithUri>(json);
+
+            Assert.Null(result.url);
```
Comment (prabirshrestha) [inline src/SimpleJson.Tests/DeserializeObjectTests.cs]: should be `IsNull`

Comment (haacked) [inline src/SimpleJson.Tests/DeserializeObjectTests.cs]: You're not in xUnit's world here Mr. @shiftkey. :stuck_out_tongue:

**Comment by shiftkey** [comment]
I've corrected that `Null` to `IsNull` to Ensure Nobody Freaks Out Any More About Unit Testing Frameworks.

I also added in a helper function which we've been using in Octokit.NET to keep things neat and tidy.

EDIT: you know what - we don't need that helper function.

**Comment by prabirshrestha** [comment]
@shiftkey your changes are now in v0.30.0 (http://www.nuget.org/packages/SimpleJson/0.30.0)

Thanks for the PR

**Pull request merged**